webhooks/services/webhooks_redis_serivce.py

```diff
@@ -319,6 +319,32 @@ def add_endorsement_event(self, event_json: str, transaction_id: str) -> None:
         if result:
             bound_logger.debug("Successfully wrote endorsement entry to redis.")
         else:
-            bound_logger.warning(
-                "Redis key for this endorsement entry is already set."
-            )  #
```

Comment:
Not sure what happened here. Accidentally remove a couple lines in a past commit!